tasks/collector.py

```diff
@@ -0,0 +1,97 @@
+import os
+import platform
+import subprocess
+import tempfile
+import urllib.request
+
+from invoke import task
+
+from tasks.go import tidy
+
+LICENSE_HEADER = """// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+"""
+
+
+@task(post=[tidy])
+def generate(ctx):
+    arch = platform.machine()
+    system = platform.system()
+    base_url = "https://github.com/open-telemetry/opentelemetry-collector/releases/download/cmd%2Fbuilder%2Fv0.103.1/"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Maybe you could extract the OCB version from the `generate` function and set it as a global for an easier maintenance
```
OCB_VERSION = "0.103.1"
```